src/figuresClasses.ts

```diff
@@ -1,19 +1,86 @@
-export interface Figure {
+enum Colors {
+  Red = 'red',
+  Green = 'green',
+  Blue = 'blue',
+}
+
+enum Shapes {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
 
+export interface Figure {
+  shape: Shapes;
+  color: Colors;
+  getArea(): number;
 }
 
 export class Triangle implements Figure {
+  shape = Shapes.Triangle;
+
+  constructor(
+    public color: Colors,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('side length should be more than zero');
+    }
 
+    if (Math.max(a, b, c) >= (a + b + c - Math.max(a, b, c))) {
+      throw new Error('You can`t build triangle with this sides');
+    }
+  }
+
+  getArea(): number {
+    const s = (this.a + this.b + this.c) / 2;
+    const area = Math.sqrt(s * (s - this.a) * (s - this.b) * (s - this.c));
+
+    return Math.floor(area * 100) / 100;
```

Comment:
You can create separate function for round down to hundredths, because you use same code 3 times